tencentcloud/resource_tc_instance_test.go

```diff
@@ -526,17 +555,17 @@ resource "tencentcloud_instance" "foo" {
   image_id      = "${data.tencentcloud_image.my_favorate_image.image_id}"
   instance_type = "${data.tencentcloud_instance_types.my_favorate_instance_types.instance_types.0.instance_type}"
 
-  system_disk_type = "CLOUD_SSD"
-  data_disks = [
-    {
-      data_disk_type = "CLOUD_SSD"
-      data_disk_size = 100
-    },
-    {
-      data_disk_type = "CLOUD_SSD"
-      data_disk_size = 100
-    }
-  ]
+  system_disk_type = "CLOUD_PREMIUM"
+  data_disks {
+    data_disk_type = "CLOUD_PREMIUM"
+    data_disk_size = 100
+    delete_with_instance = false
```

Comment:
please set it to true or remove it, otherwise every time the ci test is triggered, there will be many data disk remains, that is costly